build.go

```diff
@@ -85,6 +91,15 @@ func Build(projectPathParser PathParser, buildManager BuildManager, clock chrono
 			if err != nil {
 				return packit.BuildResult{}, err
 			}
+
+			sBom, err := sbomGenerator.Generate(context.WorkingDir)
+			if err != nil {
+				return packit.BuildResult{}, err
```

Comment:
There doesn't seem to be test coverage for this failure case

Reply:
I've added coverage.